markdown/dev/reference/api/path/addtext/en.md

```diff
@@ -7,7 +7,7 @@ The `Path.addText()` method adds text to the path.
 ## Signature
 
 ```js
-Path path.addText(string text, string className)
+Path path.addText(string text[, string className])
```

Comment:
While that `[ brackets ]` notation is rather common in man files and so on, it's not how we tend to document things in the FreeSewing docs.

Parameters that are optional typically have a default value. So I'm in favor of showing that:
```mjs
Path path.addText(string text, string className = '')
```

This makes it obvious the parameter is optional to people who are used to JavaScript, as opposed to people who are used to read man pages.